src/newlines.cpp

```diff
@@ -1945,8 +1945,8 @@ static void newlines_do_else(Chunk *start, iarf_e nl_opt)
 
 static bool is_var_def(Chunk *pc, Chunk *next)
 {
-   if (  chunk_is_token(pc, CT_DECLTYPE)
-      && chunk_is_token(next, CT_PAREN_OPEN))
+   if (  pc->Is(CT_DECLTYPE)
+      && next->Is(CT_PAREN_OPEN))
```

Comment:
need check against nulptr for both pc and next

Reply:
I've found other cases where the arguments passed into functions are not checked.
Is it preferred to always have all pointers checked, even if the function is static and used in one or two cases, and checked by the caller?

Reply:
If all the callers of the function check the pointer before the call, then we don't need to add a check here. Reason being that one of the objective of the code improvememnt effort is to make sure all Chunk pointers will never be nullptr (they will be NullChunkPtr in case). So when the improvement is completed, there will be no checks against nullptr, therefore if we know all the calls to a function are safe, then we can skip adding a check that would later been removed anyway.